e2e/e2e.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# fail on error
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# This file contatins psql views with complex types to validate and troubleshoot dbeam
+
+PSQL_USER=postgres
+PSQL_PASSWORD=mysecretpassword
+PSQL_DB=dbeam_test
+
+startPostgres() {
+  docker run --name dbeam-postgres \
+    -e POSTGRES_DB=dbeam_test \
+    -e POSTGRES_PASSWORD=mysecretpassword \
+    -v /tmp/pgdata:/var/lib/postgresql/data \
+    -p 54321:5432/tcp -d postgres:10 || true
+  # https://stackoverflow.com/questions/35069027/docker-wait-for-postgresql-to-be-running
+  docker run -it --rm --link dbeam-postgres:postgres -e PGPASSWORD=mysecretpassword postgres:10 timeout 10s bash -ic 'until psql -h postgres -U postgres dbeam_test -c "select 1"; do sleep 1; done; echo "psql up and running.."'
+  sleep 3
+  cat ./ddl.sql \
+    | docker run -i --rm --link dbeam-postgres:postgres -e PGPASSWORD=mysecretpassword postgres:10 psql -h postgres -U postgres dbeam_test
+  echo '\d' | docker run -i --rm --link dbeam-postgres:postgres -e PGPASSWORD=mysecretpassword postgres:10 psql -h postgres -U postgres dbeam_test
+}
+
+dockerClean() {
+  docker rm -f dbeam-postgres
+  #rm -r /tmp/pgdata
```

Comment:
Do we want to keep this ? `#rm -r /tmp/pgdata1

Reply:
Uncommented now. Goal is to end this script in a clean state, so next time it runs it would not break due to some bad state.